flowup-domcinTask/src/app/selectors/account.selectors.ts

```diff
@@ -0,0 +1,14 @@
+import { AppState } from '../reducers';
+import { createSelector } from '@ngrx/store';
+
+const $accountState = (state: AppState) => state.account;
+
+export const $accounts = createSelector(
+  $accountState,
+  state => state.accounts,
+);
+
+export const $loggedAccount = createSelector(
+  $accounts,
+  accounts => accounts.filter(account => account.logged)[0],
```

Comment:
```suggestion
  accounts => accounts.find(account => account.logged),
```

`find` will return `undefined` if nothing matches the predicate. Much better result than a runtime error when doing `[0]` on an empty array.